.travis.yml

```diff
@@ -114,7 +114,7 @@ matrix:
 
          - python: 3.5
            stage: Cron tests
-           env: SETUP_CMD='test --figure' ASTROPY_VERSION='dev'
+           env: SETUP_CMD='test --figure-only' ASTROPY_VERSION='dev'
```

Comment:
is this correct? surely we want all the tests including the figure ones to run with dev astropy?

Reply:
Well that test was only for the figures since we take the fig env set it up and update astropy to dev. We could leave it for all the tests.

Reply:
oh right, no problem then